electricitylci/generation_mix.py

```diff
@@ -342,19 +339,150 @@ def olcaschema_genmix(database, gen_dict, subregion=None):
                         break
                 if matching_dict is None:
                     logging.warning(
-                        f"Trouble matching dictionary for {fuelname} - {reg}"
+                        f"Trouble matching dictionary for generation mix {fuelname} - {reg}. Skipping this flow for now"
                     )
                 else:
+                    ra = exchange_table_creation_input_genmix(
+                    database_f1, fuelname
+                    )
+                    ra["quantitativeReference"] = False                    
                     ra["provider"] = {
-                        "name": matching_dict["name"],
-                        "@id": matching_dict["uuid"],
-                        "category": matching_dict["category"].split("/"),
+                       "name": matching_dict["name"],
```

Comment:
These need to be reverted - if anything else for spacing consistency within the same section.

Reply:
@jump2conclusionsmatt Matt, I created an [issue](https://github.com/USEPA/ElectricityLCI/issues/142) for this change. Due to the method used, many fuel technologies which are dropped in the generation process but show up in the generation mixes which causes a problem when imported into openLCA with blank flows. I am printing the flows that are skipped but making sure that the corresponding flows are not created.  Will this cause any problem?

Reply:
Oh I'm sorry - I didn't mean the whole thing. I just meant the changes at 349-351 - which are seemingly just spaces removed (for 3 spaces). The rest of the file uses 4 spaces that's all.